src/parser/generated_parser.ts

```diff
@@ -126,7 +128,7 @@ export class GeneratedParser extends Parser {
             (literal_3 = this.expect("**")) &&
             (c = this.expression())
         ) {
-            return pegen.seq_append_to_end(pegen.seq_append_to_end(a, b), c);
+            return pegen.seq_append_to_end(this, CHECK(pegen.seq_append_to_end(this, a, b)), c);
```

Comment:
@s-cork you'd removed this `CHECK` function but I'm not entirely sure what it does in this case

Reply:
Yeah - not sure exactly - seems to be about handling errors in C.
defined at the top of pegen.h